docs/en/operation/Admin-CLI.md

```diff
@@ -362,3 +396,158 @@ alluxio.master.audit.logging.enabled
 alluxio.master.ufs.managed.blocking.enabled
 alluxio.master.metastore.inode.inherit.owner.and.group
 ```
+
+### decommissionWorker
+
+The `decommissionWorker` command can be used to take the target workers off-line from the cluster, 
+so Alluxio clients and proxy instances stop using those workers, and therefore they can be killed or restarted gracefully.
+Note that this command will NOT kill worker processes. This command will NOT remove the cache on the workers.
+This command can be typically used for the following use cases:
+1. Perform a graceful rolling restart of all workers in the cluster, where no user requests should fail.
+2. Scale down the cluster without interrupting user I/O workflow.
+
+```shell
+$ ./bin/alluxio fsadmin decommissionWorker --addresses data-worker-0,data-worker-1 [--wait 5m] [--disable]
+```
+The arguments are explained as follows:
+
+`--address/-a` is a required argument, followed by a list of comma-separated worker addresses. Each worker address is `<host>:<web port>`.
+Unlike many other commands which specify the RPC port, we use the web port here because the command will monitor the worker's workload
+exposed at the web port. If the port is not specified, the value in `alluxio.worker.web.port` will be used. Note that `alluxio.worker.web.port`
+will be resolved from the node where this command is run.
+
+`--wait/-w` is an optional argument. This argument defines how long this command waits for the workers to become idle.
```

Comment:
How is a worker being "idle" defined? When a worker is deemed idle, what promises concerning data integrity and operational stability can the user expect and _not_ expect?

Reply:
added a paragraph for correctness